configs/repvgg/README.md

```diff
@@ -0,0 +1,46 @@
+# Repvgg: Making vgg-style convnets great again
+
+## Introduction
+
```

Comment:
Please add `<!-- [ALGORITHM] -->`, and it will be used to count algorithms in docs.